lib/util/tests/ert_util_type_vector_test.cpp

```diff
@@ -373,11 +373,21 @@ void test_equal_index() {
   int_vector_free(v3);
 }
 
-
+void test_misc() {
+  int_vector_type * v = int_vector_alloc(5, 123);
+  test_assert_int_equal( int_vector_iget(v, 2), 123 );
+  int_vector_resize(v, 20, 0);
+  test_assert_int_equal( int_vector_iget(v, 4), 123);
+  test_assert_int_equal( int_vector_iget(v, 9), 0 );
+  test_assert_int_equal( int_vector_iget(v, 19), 0);
+  //int_vector_iset(v, 10, 66);
+  //test_assert_int_equal( int_vector_iget(v, 10), 66);
+}
 
 int main(int argc , char ** argv) {
+  test_misc();
 
-  int_vector_type * int_vector = int_vector_alloc( 0 , 99);
+  /*int_vector_type * int_vector = int_vector_alloc( 0 , 99);
```

Comment:
Looks like a large block of code has been commented out?

Reply:
A mistake, uncommented.